<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CalibApp</class>
 <widget class="QMainWindow" name="CalibApp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Camera Calibration</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>11</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QScrollArea" name="ImageFrame" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>13</hsizetype>
        <vsizetype>13</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="PixmapWidget" name="outputImage" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="posX" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="posY" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="posZ" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rotX" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rotY" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rotZ" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1_3" >
        <property name="text" >
         <string>Focal Distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FocalDistance" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>106</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="VideoServerLabel" >
        <property name="text" >
         <string>Video Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="VideoServer" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>gin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="IndexLabel" >
        <property name="text" >
         <string>Index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cameraIndex" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AcquireButton" >
        <property name="text" >
         <string>Acquire</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Calibrate" >
        <property name="text" >
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="AutoLabel" >
        <property name="text" >
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ProcessButton" >
        <property name="text" >
         <string>Process Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StageButton" >
        <property name="text" >
         <string>Show Stage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="StageBox" />
      </item>
      <item>
       <widget class="QPushButton" name="OriginalButton" >
        <property name="text" >
         <string>Original</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="ManualLabel" >
        <property name="text" >
         <string>Manual</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlaceButton" >
        <property name="text" >
         <string>Place Points</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PointLabel" >
        <property name="text" >
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PointBox" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>26</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PointXLabel" >
        <property name="text" >
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PointX" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PointYLabel" >
        <property name="text" >
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PointY" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PointZLabel" >
        <property name="text" >
         <string>z:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PointZ" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="editMenu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="editUndoAction" />
    <addaction name="editRedoAction" />
    <addaction name="separator" />
    <addaction name="editCutAction" />
    <addaction name="editCopyAction" />
    <addaction name="editPasteAction" />
    <addaction name="separator" />
    <addaction name="editFindAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpContentsAction" />
    <addaction name="helpIndexAction" />
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
   </widget>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction" />
    <addaction name="fileLoad_3D_PointsAction" />
    <addaction name="fileLoad_2D_PointsAction" />
    <addaction name="fileSave_2D_PointsAction" />
    <addaction name="fileSaveAction" />
    <addaction name="fileSaveAsAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="editMenu" />
   <addaction name="helpMenu" />
  </widget>
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset>image0</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="iconText" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset>image1</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Image...</string>
   </property>
   <property name="iconText" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAction" >
   <property name="icon" >
    <iconset>image2</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="iconText" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="icon" >
    <iconset>image3</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save As</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="filePrintAction" >
   <property name="icon" >
    <iconset>image4</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="editUndoAction" >
   <property name="icon" >
    <iconset>image5</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="iconText" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="editRedoAction" >
   <property name="icon" >
    <iconset>image6</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="iconText" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="editCutAction" >
   <property name="icon" >
    <iconset>image7</iconset>
   </property>
   <property name="text" >
    <string>&amp;Cut</string>
   </property>
   <property name="iconText" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="editCopyAction" >
   <property name="icon" >
    <iconset>image8</iconset>
   </property>
   <property name="text" >
    <string>C&amp;opy</string>
   </property>
   <property name="iconText" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="editPasteAction" >
   <property name="icon" >
    <iconset>image9</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="iconText" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="editFindAction" >
   <property name="icon" >
    <iconset>image10</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find...</string>
   </property>
   <property name="iconText" >
    <string>Find</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="helpContentsAction" >
   <property name="text" >
    <string>&amp;Contents...</string>
   </property>
   <property name="iconText" >
    <string>Contents</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpIndexAction" >
   <property name="text" >
    <string>&amp;Index...</string>
   </property>
   <property name="iconText" >
    <string>Index</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="fileLoad_3D_PointsAction" >
   <property name="text" >
    <string>Load &amp;3D Points...</string>
   </property>
   <property name="iconText" >
    <string>Load 3D Points</string>
   </property>
  </action>
  <action name="fileLoad_2D_PointsAction" >
   <property name="text" >
    <string>Load &amp;2D Points...</string>
   </property>
   <property name="iconText" >
    <string>Load 2D Points</string>
   </property>
  </action>
  <action name="fileSave_2D_PointsAction" >
   <property name="text" >
    <string>Save &amp;2D Points...</string>
   </property>
   <property name="iconText" >
    <string>Save 2D Points</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QScrollArea</class>
   <extends>QFrame</extends>
   <header>QScrollArea</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>PixmapWidget</class>
   <extends>QLabel</extends>
   <header>PixmapWidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
